test/skimNtuple2018.cpp

```diff
@@ -2309,6 +2309,91 @@ int main (int argc, char** argv)
       float idAndIsoAndFakeSF_deep_pt = 1.0; // use this for DeepTauV2p1 pt dependent + e/mu->tauh fake SF
       float fakeRateSF_MVA  = 1.0;           // use this for e/mu->tauh fake SF MVA
       float fakeRateSF_deep = 1.0;           // use this for e/mu->tauh fake SF DeepTau
+       
+      float idAndIsoAndFakeSF_deep_pt_tau_pt20to25_up   = 1.0;
+      float idAndIsoAndFakeSF_deep_pt_tau_pt25to30_up   = 1.0;
+      float idAndIsoAndFakeSF_deep_pt_tau_pt30to35_up   = 1.0;
+      float idAndIsoAndFakeSF_deep_pt_tau_pt35to40_up   = 1.0;
+      float idAndIsoAndFakeSF_deep_pt_tau_pt40toInf_up  = 1.0;
+      
+      float idAndIsoAndFakeSF_deep_pt_tau_pt20to25_down = 1.0;
+      float idAndIsoAndFakeSF_deep_pt_tau_pt25to30_down = 1.0;
+      float idAndIsoAndFakeSF_deep_pt_tau_pt30to35_down = 1.0;
+      float idAndIsoAndFakeSF_deep_pt_tau_pt35to40_down = 1.0;
+      float idAndIsoAndFakeSF_deep_pt_tau_pt40toInf_down= 1.0;
+      
+      float idAndIsoAndFakeSF_deep_mu_eta0to0p4_up      = 1.0;
+      float idAndIsoAndFakeSF_deep_mu_eta0p4to0p8_up    = 1.0;
+      float idAndIsoAndFakeSF_deep_mu_eta0p8to1p2_up    = 1.0;
+      float idAndIsoAndFakeSF_deep_mu_eta1p2to1p7_up    = 1.0;
+      float idAndIsoAndFakeSF_deep_mu_eta1p7toInf_up    = 1.0;
+      
+      float idAndIsoAndFakeSF_deep_mu_eta0to0p4_down    = 1.0;
+      float idAndIsoAndFakeSF_deep_mu_eta0p4to0p8_down  = 1.0;
+      float idAndIsoAndFakeSF_deep_mu_eta0p8to1p2_down  = 1.0;
+      float idAndIsoAndFakeSF_deep_mu_eta1p2to1p7_down  = 1.0;
+      float idAndIsoAndFakeSF_deep_mu_eta1p7toInf_down  = 1.0;
+      
+      float idAndIsoAndFakeSF_deep_ele_barrel_up     = 1.0;
```

Comment:
maybe it's better to use the names suggested on the twiki: `idAndIsoAndFakeSF_etauFR_barrel_up`...